snakemake/subworkflow.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from snakemake.logging import logger
+from snakemake.exceptions import WorkflowError
+
+
+def subworkflow(
+    path,
+    basedir,
+    input,
+    output,
+    params,
+    wildcards,
+    threads,
+    resources,
```

Comment:
are resources enforced in sub-workflows?

Reply:
Is there something I'd have to do manually? When executing on a cluster, the cluster profiles should take care of this, shouldn't they?